src/test/java/org/broadinstitute/consent/http/service/DataAccessRequestServiceTest.java

```diff
@@ -789,4 +803,126 @@ public boolean matches(String argument) {
       return argument.length() > 2;
     }
   }
+
+  @Test
+  void testSendReminderMessage() {
+    Election election = new Election();
+    election.setElectionId(RandomUtils.nextInt());
+    election.setReferenceId(UUID.randomUUID().toString());
+    election.setElectionType(ElectionType.DATA_ACCESS.getValue());
+    when(electionDAO.findElectionWithFinalVoteById(any())).thenReturn(election);
+
+    Vote vote = new Vote();
+    vote.setVoteId(RandomUtils.nextInt());
+    vote.setElectionId(election.getElectionId());
+    when(voteDAO.findVoteById(any())).thenReturn(vote);
+
+    DarCollection collection = new DarCollection();
+    collection.setDarCollectionId(RandomUtils.nextInt());
+    collection.setDarCode("DAR-12345");
+    when(darCollectionDAO.findDARCollectionByReferenceId(any())).thenReturn(collection);
+
+    User user = new User();
+    user.setDisplayName(RandomStringUtils.randomAlphanumeric(10));
+    user.setEmail(RandomStringUtils.randomAlphanumeric(10));
+    when(userDAO.findUserById(any())).thenReturn(user);
+
+    initService();
+    assertDoesNotThrow(() -> service.sendReminderMessage(vote.getVoteId()));
```

Comment:
If these are also copied over tests, I guess that's OK, just note that these are also not really testing our code's behavior. For this test, to improve it, add to initService():
```
    config.getServicesConfiguration().setLocalURL("local_url/");
```

And then add here:
```
    verify(emailService)
        .sendReminderMessage(
            user,
            vote,
            collection.getDarCode(),
            election.getElectionType(),
            "local_url/dar_collection/" + collection.getDarCollectionId());
    verify(voteDAO).updateVoteReminderFlag(vote.getVoteId(), true);
```